canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html

```diff
@@ -16,17 +16,19 @@
   {% endif %}
 {% endblock stylesheets %}
 
-{% block content %}
 
+{% block breadcrumb %}
+<nav>
+    <h3>
```

Comment:
(that would also mean there's no need for the new h3 styling added to manage_people.css)